service/controller/legacy/v29patch1/templates/cloudformation/tccp/record_sets.go

```diff
@@ -47,7 +47,7 @@ const RecordSets = `
       Name: '{{ $v.EtcdDomain }}.'
       HostedZoneId: !Ref 'InternalHostedZone'
       Type: A
-  EtcdRecordSet:
+  EtcdExternalRecordSet:
     Type: AWS::Route53::RecordSet
```

Comment:
I have realized it was breaking the upgrade as it was trying to update the record set with same name but in different zone

```
[Tried to create resource record set [name='etcd.0cuut.k8s.ginger.eu-central-1.aws.gigantic.io.', type='A'] but it already exists]
```
We need to keep the internal the same